src/components/layout/Header/Header.js

```diff
@@ -1,17 +1,19 @@
 import { NavbarDesktop } from "../../common/NavbarDesktop";
 import { Logo, MenuButton, CartSidebarView, SearchForm } from "../../ui";
 import { Heading, ActionsContainer, NavDeskContainer } from "./styled";
+import { NavbarSmallScreen } from "./../../common";
 
-const Header = ({ handleNavigate }) => {
+const Header = () => {
   return (
     <Heading>
-      <Logo onClick={() => handleNavigate(true)} />
+      <Logo />
       <ActionsContainer>
         <NavDeskContainer>
-          <NavbarDesktop onClick={() => handleNavigate(true)} />
+          <NavbarDesktop />
```

Comment:
what props does NavbarDesktop has? in this path "./common/NavbarDesktop/NavbarDesktop.js" the component has props but what values are? why this component does not get any of them?

Reply:
the implementation is wrong, i fix it